app/views/sessions.jade

```diff
@@ -0,0 +1,44 @@
+.sessions
+	h2 Sessions
+
+	h3 Keynotes
+	p Don't Call Me Sweetie: How to Build Confidence and Create a Great Workplace for Women: Kristin Smith
+	p Career Advancement: The System is Rigged: Davy Stevenson
+
+	h3 Lightning Talks
+	p Telling Stories, Building Technologies
+	p Work Outreach Balance
+	p In '69: No Experience Necessary
+	p Agile: A Cautionary Tale
+	p Keep Yourself - Change Your Company
+	p Trans-inclusivity and Trans Women’s Tech Experience
+
+	h3 Breakout Sessions - Career Advancement
+	p How to “Do” Talks and Workshops 
+	p Getting off the Freelance Roller Coaster: Learn a System for Selling
+	p Getting Started in Open Source
+	p The Authentic You
+	p Leading with Grace
+	p Up Your Game Through Coaching
+	p Imposter Syndrome
+	p Practical Advice for Establishing and Engineering Your Career
+	p How to Impress Your Future Employer
+	p The Art of Interviewing: Learning to Tell Your Story
+	p Create a To-Do for Diversity Guide for Employers in Tech
+	p Getting Started with Full Stack: Control Your Apps from Back to Front
+	p Negotiation (name TBA)
+
+	h3 Tech Workshops
+	p 
```

Comment:
When I made a second commit after setting a pull request on the first commit, the second one was automatically added to the first. 

I wasn't sure why or if I had done something wrong.... I kind of asked about it in the slack.